document_url_google_drive/readme/USAGE.md

```diff
@@ -0,0 +1,8 @@
+To use this module, you need to:
+
+#. Open the form view of an object (Example: Customer Invoice INV/2019/0007). #. Go to
+the chatter and click on the attached icon. #. Click **Add GDrive link**. #. Fill the
+wizard fields and click on Add button. #. In the open window, select the files you need
+and press the select button. #. You will see a new **URL attachment** in the set of
+attachments related to the object. #. In order to log in under another google user,
+click on the logout icon located after **Add GDrive link**
```

Comment:
Hi @ivs-cetmix . I guess this will do.

```suggestion
1. Open the form view of an object (Example: Customer Invoice INV/2019/0007).
1. Go to the chatter and click on the attached icon.
1. Click **Add GDrive link**.
1. Fill the wizard fields and click on Add button.
1. In the open window, select the files you need and press the select button.
1. You will see a new **URL attachment** in the set of attachments related to the object.
1. In order to log in under another google user, click on the logout icon located after **Add GDrive link**
```

Reply:
> Hi @ivs-cetmix . I guess this will do.

Thank you @yostashiro ! I have merged your suggestions and also added some screenshots to make the documentation clearer